src/dodal/devices/bimorph_mirrors/CAENels_bimorph_mirror_16_channel.py

```diff
@@ -0,0 +1,174 @@
+from ophyd import Component, EpicsSignal, EpicsSignalRO
+
+from dodal.devices.bimorph_mirrors.CAENels_bimorph_mirror_8_channel import (
+    CAENelsBimorphMirror8Channel,
+)
+
+
+class CAENelsBimorphMirror16Channel(CAENelsBimorphMirror8Channel):
```

Comment:
As we talked about on slack, it will be nicer to have a class for a channel which does all the repeated stuff. I had a similar problem a while ago, see the code in https://github.com/DiamondLightSource/dodal/tree/main/src/dodal/devices/xspress3_mini. `Xspress3Mini` has a channel_1 attribute, and could easily have other channels added in a single line